src/EnergyPlus/DXCoils.cc

```diff
@@ -14363,6 +14363,10 @@ void CalcTwoSpeedDXCoilStandardRating(EnergyPlusData &state, int const DXCoilNum
                                 thisDXCoil.Name));
         return;
     }
+    bool saveTurnFansOn = state.dataHVACGlobal->TurnFansOn;
+    bool saveTurnFansOff = state.dataHVACGlobal->TurnFansOff;
+    state.dataHVACGlobal->TurnFansOn = true; // enable fans, will override fan availability schedule if needed
+    state.dataHVACGlobal->TurnFansOff = false;
```

Comment:
Since the calls to the fan hard coded the avail status flags, which turns out to be needed (because I had unit test and sizing diffs without this) in case a fan availability schedule does not allow the fan to run on this date, this was added to enable the fans. Save what the flags were, set them to allow the fan to run, and then reset to what these flags were before calling this function.